plugins/inputs/deprecations.go

```diff
@@ -0,0 +1,47 @@
+package inputs
+
+import "github.com/influxdata/telegraf"
+
+var Deprecations = map[string]telegraf.DeprecationInfo{
+	"cassandra": {
+		Since:  "1.7",
```

Comment:
Yeah I can do this, but it will just increase the "writing work". :-) Semver anyway says that you patches _only_ fix bugs, so deprecating a plugin in anything not `x.y.0` is ill-posed I think. Anyway, I can change this if you like, just let me know.

Reply:
Totally understand the concern, but I would prefer we are explicit as we are with tagging and other places like build_version.txt.

Reply:
I agree with that, it doesn't hurt and doesn't need change of code and it becomes more specific. Otherwise one might think it means that a plugin/option got deprecated 'somewhere' in `x.y` release.